.travis.yml

```diff
@@ -3,9 +3,7 @@ node_js:
   - 0.10
 
 install:
-  - npm install -g bower grunt grunt-cli
-  - npm install grunt-jscs grunt-contrib-jshint grunt-htmlhint grunt-contrib-csslint
-  - npm install grunt-jinja grunt-jscs grunt-shell grunt-jstestdriver-phantomjs grunt-closurecompiler
+  - npm install
```

Comment:
Looks like it passed, so this likely doesn't matter, but the old one installed grunt/grunt-cli with the global flag, the new way doesn't. As long as that is ok, lgtm.